modules/store/testing/src/mock_store.ts

```diff
@@ -47,6 +48,7 @@ export class MockStore<T> extends Store<T> {
     super(state$, actionsObserver, reducerManager);
     this.resetSelectors();
     this.state$.next(this.initialState);
+    this.lastState = this.initialState;
```

Comment:
what about calling  `setState(this.initialState)` here?

Reply:
Yep